packages/devtools/lib/src/timeline/timeline_protocol.dart

```diff
@@ -532,6 +533,22 @@ class TimelineEvent {
     }
   }
 
+  /// Returns the depth of this TimelineEvent tree, including [this].
+  int getDepth() {
+    int maxDepth(TimelineEvent root) {
+      if (root == null) {
+        return 0;
+      }
+      int maximum = 0;
```

Comment:
nit: instead of calling this maximum, maybe just call it depth?

Reply:
done.